templates/web/bromley/footer_extra.html

```diff
@@ -38,4 +38,17 @@ <h5>Social</h5>
             </div><!--copyright-->
         </div><!--column-->
     </div><!--footer-container-->
-</footer>
\ No newline at end of file
+</footer>
+
+<div class="societyworks-footer">
+    <div class="container">
+        <p>
+            Powered by
+          [% IF bodyclass == 'waste' %]
```

Comment:
Ah, the problem here is that a waste report page isn't a bodyclass==waste page. Looking at other templates, looks like we have elsewhere used `bodyclass == 'waste' OR (problem AND problem.cobrand_data == 'waste')` which I think would also work here.

Reply:
Ah! Thanks for spotting that. It’s a shame we don’t have a simpler "is this part of the waste site or not" flag. Oh well, probably something for another day. I’ve pushed a fixup commit with the change you suggested.